news.html

```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
           <h1 class="page-header">Interledger News</h1>
 
           <p class="intro">Here you can find the latest Interledger developments and community announcements.</p>
 
 
          <ul class="list-group">
+         <li class="list-group-item">
+             <span class="label label-default label-pill pull-right">Oct 24</span>
+             Chain releases their <a target="_blank" href="https://chain.com/docs/protocol/papers/whitepaper#9-interoperability">Chain Protocol Whitepaper</a> citing Interledger for cross-chain interoperability.
```

Comment:
How about just "Chain Protocol Whitepaper cites Interledger for cross-chain interoperability"?